packages/bot-skeleton/src/services/tradeEngine/trade/index.js

```diff
@@ -130,19 +133,28 @@ export default class TradeEngine extends Balance(Purchase(Sell(OpenContract(Prop
                     }, 1500);
                 }
                 if (data.msg_type === 'authorize') {
-                    this.accountInfo = data;
+                    const { authorize = {} } = data;
+                    this.accountInfo = authorize;
                     this.token = token;
 
                     // Only subscribe to balance in browser, not for tests.
                     if (document) {
-                        doUntilDone(() => this.api.send({ balance: 1, subscribe: 1 })).then(r => {
+                        doUntilDone(() => this.api.send({ balance: 1 })).then(r => {
                             this.balance = Number(r.balance.balance);
                             resolve();
                         });
```

Comment:
```suggestion
                        }).catch(err => {
                            this.$scope.observer.emit('Error', err);
                        });
```

Reply:
do until done has its own error handler so we don't need to put catch statement here.
I did this in our other project bot.deriv.com and ended up getting more trackjs issues.
So I think it is not a good idea to add a catch statement here. @maryiafrantsava-binary